Gui/python/ArduinoWidget.py

```diff
@@ -71,114 +73,80 @@ def createArduino(self):
         self.UseArduino.clicked.connect(self.frozeArduinoPanel)
         self.ReleaseArduino = QPushButton("&Release")
         self.ReleaseArduino.clicked.connect(self.releaseArduinoPanel)
+        self.TempScriptUpload = QPushButton("&Upload script")
+        self.TempScriptUpload.clicked.connect(self.TemScriptUploder)
         self.ReleaseArduino.setDisabled(True)
-
         self.ArduinoBox.addWidget(self.ArduinoStatusLabel)
         self.ArduinoBox.addWidget(self.ArduinoCombo)
         self.ArduinoBox.addWidget(self.ArduinoBaudRate)
         self.ArduinoBox.addWidget(self.ArduinoBRCombo)
-        # self.ArduinoBox.addWidget(self.ArduinoValues)
         self.ArduinoBox.addStretch(1)
         self.ArduinoBox.addWidget(self.UseArduino)
         self.ArduinoBox.addWidget(self.ReleaseArduino)
+        self.ArduinoBox.addWidget(self.TempScriptUpload)
         self.ArduinoGroup.setLayout(self.ArduinoBox)
         self.mainLayout.addWidget(self.ArduinoGroup, 0, 0)
 
         self.ArduinoMeasureValue = QLabel()
         self.mainLayout.addWidget(self.ArduinoMeasureValue, 1, 0)
+    
 
     def listResources(self):
-        self.ResourcesManager = visa.ResourceManager("@py")
+        self.ResourcesList = []
```

Comment:
This function was removed? Do we not actually need it?

Reply:
listResources() is needed. It is used for getting Arduino communication port from site_settings.